src/cart/crt_init.c

```diff
@@ -62,52 +100,19 @@ crt_lib_fini(void)
 static void
 dump_envariables(void)
 {
-	int	i;
-	char	*val;
-	char    *envars[] = {"D_PROVIDER",
-			     "D_INTERFACE",
-			     "D_DOMAIN",
-			     "D_PORT",
-			     "CRT_PHY_ADDR_STR",
-			     "D_LOG_STDERR_IN_LOG",
-			     "D_LOG_SIZE",
-			     "D_LOG_FILE",
-			     "D_LOG_FILE_APPEND_PID",
-			     "D_LOG_MASK",
-			     "DD_MASK",
-			     "DD_STDERR",
-			     "DD_SUBSYS",
-			     "CRT_TIMEOUT",
-			     "CRT_ATTACH_INFO_PATH",
-			     "OFI_PORT",
-			     "OFI_INTERFACE",
-			     "OFI_DOMAIN",
-			     "CRT_CREDIT_EP_CTX",
-			     "CRT_CTX_SHARE_ADDR",
-			     "CRT_CTX_NUM",
-			     "D_FI_CONFIG",
-			     "FI_UNIVERSE_SIZE",
-			     "CRT_ENABLE_MEM_PIN",
-			     "FI_OFI_RXM_USE_SRX",
-			     "D_LOG_FLUSH",
-			     "CRT_MRC_ENABLE",
-			     "CRT_SECONDARY_PROVIDER",
-			     "D_PROVIDER_AUTH_KEY",
-			     "D_PORT_AUTO_ADJUST",
-			     "D_POLL_TIMEOUT",
-			     "D_LOG_FILE_APPEND_RANK",
-			     "D_QUOTA_RPCS",
-			     "D_POST_INIT",
-			     "D_POST_INCR",
-			     "DAOS_SIGNAL_REGISTER"};
+	int                i;
+	char              *val;
```

Comment:
These are presumably a clang-format warning now, however...

Reply:
I am on it ;)